apps/labs/posts/implement-lapack-routines-in-stdlib.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Adding support for LAPACK routines in stdlib"
+authors: [pranav-goswami]
+published: September 30, 2024
+description: "Implementing LAPACK routines for numerical computation in web applications"
+category: [stdlib, mathematics, linear-algebra, numerical-computing]
```

Comment:
```suggestion
category: [stdlib, mathematics, linear-algebra, numerical-computing, Internship]
```

Can you check that these categories are valid? The blogging handbook (and the repo README) have instructions on how to check that. Thanks!

Reply:
I added those in `apps/labs/posts/categories.json`.